source/appModules/explorer.py

```diff
@@ -299,3 +300,16 @@ def event_gainFocus(self, obj, nextHandler):
 			return
 
 		nextHandler()
+
+	def isGoodUIAWindow(self,hwnd):
+		# #9204: In build 18305 and later, it is the shell that raises window open event for emoji panel.
+		if winVersion.winVersion.build > 17763 and winUser.getClassName(hwnd) == "ApplicationFrameWindow":

```

Comment:
Just to make sure, why do you have to mark this (i.e. ApplicationFrameWindow) as a good UIA window? Doesn't it have a native UIA implementation? Given that an UIA event is fired on it, I'd say it does have one?